package.json

```diff
@@ -39,7 +39,7 @@
     "@babel/core": "^7.17.5",
```

Comment:
Run `yarn upgrade --latest` as this will force to upgrade a version even if it is "compatible version"
In other words, currently we have @bsv/sdk in version `^1.0.33` but the newest one is `1.0.36`, if you run the command above, it will set the version to `^1.0.36` even though it is still "in the scope" of `^` sign.

Reply:
done